README.md

```diff
@@ -1 +1,19 @@
-# Weather-App-JS
\ No newline at end of file
+# Weather-App-JS
+
+The aim of this project was to design and build a weather forecasting site using ES modules, webpack and the OpenWeather API [The Odin Project](https://www.theodinproject.com/courses/javascript/lessons/weather-app).
+
+### How to Use
+
+* Open the terminal and clone the repo 
+```
+git clone https://github.com/KerronKing/Weather-App-JS
```

Comment:
README file not detailed enough. (you can add steps to git clone your repo, necessary installation and live link)
checkout out the below link for help, free to follow the template from the pathwright
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/